libnetwork/datastore/mockstore_test.go

```diff
@@ -88,6 +88,13 @@ func (s *MockStore) AtomicDelete(key string, previous *store.KVPair) error {
 	return nil
 }
 
+// Delete deletes a value at "key". Unlike AtomicDelete it doesn't check
+// whether the deleted key is at a specific version before deleting.
+func (s *MockStore) Delete(key string) error {
```

Comment:
MockStore's Delete should behave like the boltdb implementation: return `store.ErrKeyNotFound` if delete would be a no-op.